package.json

```diff
@@ -88,6 +136,10 @@
     "redux-persist": "6.0.0",
     "redux-thunk": "2.4.2",
     "regenerator-runtime": "0.13.10",
+    "rehype-stringify": "9.0.3",
```

Comment:
This is an awful lot of extra dependencies for just generating Markdown. I'm going to need to look through those in some more detail, but are you sure they're all needed?

Reply:
the problem is that we need to go through a few steps to convert the data in the editor to MD.

HTML => HAST => MDAST => MD 

and the other way around. Although I am currently not using that functionality

Reply:
fixed